packages/core/src/types.ts

```diff
@@ -1717,6 +1718,7 @@ export interface ActorRef<TEvent extends EventObject, TSnapshot = any>
   // TODO: should this be optional?
   start?: () => void;
   getSnapshot: () => TSnapshot | undefined;
+  getPersistedState?: () => any;
```

Comment:
`any` definitely shouldn't be used here, I'm not yet sure though at which level we should care about this. Perhaps only the root `actorRef` could expose this? But then... Its own `TPersistedState` should probably encode the persisted state types of all of their children (recursively). 

But maybe this is not a concern for now. We can reexamine this when revamping the types as a whole

Reply:
Added a todo for this